src/modules/0.0.21/aws_elasticache/index.ts

```diff
@@ -17,7 +17,7 @@ class CacheClusterMapper extends MapperBase<CacheCluster> {
   equals = (a: CacheCluster, b: CacheCluster) =>
```

Comment:
don't we need to add the region to the equals method too?

Reply:
Oh, yes. It probably didn't do anything in the update path here.